bindings/python/IK/tests/model.urdf

```diff
@@ -0,0 +1,3414 @@
+<robot name="iCub">
```

Comment:
I'm not a big fan of bringing the urdf model inside the repo. As far as I understood you need it for the test. In C++ we use random models generated by iDynTree. 

https://github.com/dic-iit/bipedal-locomotion-framework/blob/0c055db96d25fa94d588626d8df8a375a0b04c19/src/IK/tests/QPInverseKinematicsTest.cpp#L225-L226


In iDynTree the model class  has already pybind11 bindings so I think it shouldn't be so complex to add this method https://github.com/robotology/idyntree/blob/master/bindings/pybind11/idyntree_model.cpp

Reply:
Could be addressed by https://github.com/dic-iit/bipedal-locomotion-framework/pull/303#discussion_r631066529

Reply:
Regarding the urdf model, I took inspiration from the [FloatingBaseEstimators/tests](https://github.com/dic-iit/bipedal-locomotion-framework/tree/0c055db96d25fa94d588626d8df8a375a0b04c19/bindings/python/FloatingBaseEstimators/tests) which already contained an urdf for testing.

In order to address this comment, is it ok to modify the constructor of the [KinDynComputationsDescriptor](https://github.com/dic-iit/bipedal-locomotion-framework/blob/0c055db96d25fa94d588626d8df8a375a0b04c19/src/Estimators/src/ModelComputationsHelper.cpp#L27) class so that it can also take a model to be directly loaded at https://github.com/dic-iit/bipedal-locomotion-framework/blob/0c055db96d25fa94d588626d8df8a375a0b04c19/src/Estimators/src/ModelComputationsHelper.cpp#L74 or do you suggest different solutions?

Reply:
A possibility is to get the urdf from internet using `wget` python package 
https://pypi.org/project/wget/

Reply:
If we need to add an external dependency, at this point we could add `gym-ignition-models` that has high-level functions to return either fixed or floating-base models in both URDF and SDF formats.

https://github.com/robotology/gym-ignition-models#usage

Reply:
I agree with @diegoferigo we can add `gym-ignition-models` for the test

Reply:
Just remember that the icub model in that repo is modified to be compatible with Ignition Gazebo (i.e. no yarp plugins, and other minor modifications). However, for the sake of these kind of tests, it will be ok.

Reply:
After installing `gym-ignition-models` via pip as suggested in https://github.com/robotology/gym-ignition-models#installation, I removed the urdf and updated the test via https://github.com/dic-iit/bipedal-locomotion-framework/pull/303/commits/2f412b8e4a3e0714de16c062a71bdb7605430dd9.

Should we handle the additional dependency by automatically installing the package or just adding a "please install `gym-ignition-models`" in the [README section dedicated to the python bindings](https://github.com/dic-iit/bipedal-locomotion-framework#snake-python)?

Reply:
xref https://github.com/robotology/icub-models/issues/91

Reply:
For the time being, as discussed with @GiulioRomualdi and @traversaro, we do not add external dependecies and load the urdf models through [urllib.request](https://docs.python.org/3/library/urllib.request.html). Implemented in a2bbd0859a36cdd0b8d2700f0b245094b09817a6.

Reply:
Nit: no need to close the tmp file, the garbage collector does it automatically when the object goes out of scope

Reply:
I was not sure that was the case, done via 7e57363665a649ad7ba97f209189c724af0f1165.